rskj-core/src/test/java/co/rsk/mine/MinerServerTest.java

```diff
@@ -763,6 +778,68 @@ public void secondBuildBlockToMineTurnsNotifyFlagOff() {
         }
     }
 
+    @Test
+    public void extraDataNotInitializedWithClientData() {
+        MinerServer minerServer = new MinerServerImpl(
+                config,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                new BuildInfo("cb7f28e", "master"),
+                ConfigUtils.getDefaultMiningConfig()
+        );
+
+        byte[] extraData = minerServer.getExtraData();
```

Comment:
Can we test this after building a real block?

Reply:
Sounds good, will add this to the backlog for a next release since it's not major and we need to advance with merging this PR soon.